doc/source/security/using-amz-kms.rst

```diff
@@ -0,0 +1,135 @@
+.. _using-amz-kms:
+
+================================================================================
+Using the Amazon Key Management Service (AWS KMS)
+================================================================================
+
+This feature is **technical preview** quality.
+
+**Percona Server for MySQL** 8.0.28-20 adds support for the `Amazon Key
+Management Server (AWS KMS) <https://aws.amazon.com/kms/>`__. The AWS server
+generates the keyring material and uses a local file for key storage.
```

Comment:
Which material? The keyring keys are generated by mysql, as with other keyrings. AWS only encrypts this keyring data, it doesn't generate it.

Reply:
Rewritten